service/version_bundles.go

```diff
@@ -0,0 +1,113 @@
+package service
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func NewVersionBundles() []versionbundle.Bundle {
+	return []versionbundle.Bundle{
+		{
+			Changelogs: []versionbundle.Changelog{
+				{
+					Component:   "calico",
+					Description: "Calico version updated.",
+					Kind:        "changed",
+				},
+				{
+					Component:   "docker",
+					Description: "Docker version updated.",
+					Kind:        "changed",
+				},
+				{
+					Component:   "etcd",
+					Description: "Etcd version updated.",
+					Kind:        "changed",
+				},
+				{
+					Component:   "kubedns",
+					Description: "KubeDNS version updated.",
+					Kind:        "changed",
+				},
+				{
+					Component:   "kubernetes",
+					Description: "Kubernetes version updated.",
+					Kind:        "changed",
+				},
+				{
+					Component:   "nginx-ingress-controller",
+					Description: "Nginx-ingress-controller version updated.",
+					Kind:        "changed",
+				},
+			},
+			Components: []versionbundle.Component{
+				{
+					Name:    "calico",
+					Version: "2.6.2",
+				},
+				{
+					Name:    "docker",
+					Version: "1.12.6",
+				},
+				{
+					Name:    "etcd",
+					Version: "3.2.7",
+				},
+				{
+					Name:    "kubedns",
+					Version: "1.14.5",
+				},
+				{
+					Name:    "kubernetes",
+					Version: "1.8.1",
+				},
+				{
+					Name:    "nginx-ingress-controller",
+					Version: "0.9.0",
+				},
+			},
+			Dependencies: []versionbundle.Dependency{},
+			Deprecated:   false,
+			Name:         "aws-operator",
+			Time:         time.Date(2017, time.November, 22, 16, 9, 0, 0, time.UTC),
+			Version:      "0.1.0",
+			WIP:          true,
+		},
+		{
+			Changelogs: []versionbundle.Changelog{},
```

Comment:
This should actually make the operator fail on startup. With each version bundle at least one changelog is required. We should have strong discipline with this as well. Lets just point out what changed and chose an appropriate component to refer to. I think this could even refer to cloud formation as component.

Reply:
Ack. There was an error due to the missing change log. I've added cloud formation as a component as that is nice.